exercises/strain/strain_test.go

```diff
@@ -19,6 +19,14 @@ import (
 	"testing"
 )
 
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
```

Comment:
ah I've been doing it the other way I believe. Making it the first *Test function but leaving any functions used in tests before them.